rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -11827,15 +11819,14 @@ void opcode_0404(const ScriptArguments& args) {
 }
 
 /**
-    @brief enable_phone %1d%
+    @brief enable_payphone %1d%
 
     opcode 0405
-    @arg phone Handle
+    @arg payphone Handle
 */
-void opcode_0405(const ScriptArguments& args, const ScriptPhone phone) {
-    RW_UNIMPLEMENTED_OPCODE(0x0405);
-    RW_UNUSED(phone);
+void opcode_0405(const ScriptArguments& args, const ScriptPayphone payphone) {
     RW_UNUSED(args);
+    payphone->enable();
```

Comment:
also

Reply:
Pointer to the object is unchangeable, not the object itself, object method declaration implies that object can be mutated (no const).